### PR TITLE
feat(): add volume slider; reduce initial volume 35 -> 25

### PR DESCRIPTION
**Pull request by jacobnisnevich**
image

image

#683

### CODE REVIEW

File: src/common/components/game/GameArea.js [hunk 1]
```diff
@@ -161,9 +164,9 @@ export default class GameArea extends React.Component {
     const {
       attack, bluePieces, currentTurn, eventQueue, gameOptions, gameOver, history, isAttackHappening,
       isMyTurn, isPractice, isSandbox, isSpectator, isTutorial, message, orangePieces, player, playingCardType,
-      selectedTile, sfxQueue, status, target, tutorialStep, usernames, winner,
+      selectedTile, sfxQueue, status, target, tutorialStep, usernames, winner,  volume,
```
Comment (AlexNisnevich) [inline src/common/components/game/GameArea.js]: nit: think there's an extra space here

File: src/common/actions/game.ts [hunk 2]
```diff
@@ -132,3 +133,10 @@ export function addCardToTopOfDeck(player: w.PlayerColor, card: w.Card): w.Actio
     payload: { player, card }
   };
 }
+
+export function setVolume(volume: number): w.Action {
+  return {
+    type: SET_VOLUME,
+    payload: { volume }
+  }
+}
```
Comment (AlexNisnevich) [inline src/common/actions/game.ts]: nit: newline

File: src/common/components/game/VolumeControl.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    width: 36
+  },
+  iconRootSelected: {
+    border: '2px solid #aaaaaa',
+    backgroundColor: '#444444',
+    borderRadius: '50%',
+    padding: 0,
+    height: 36,
+    width: 36,
+    color: '#ffffff !important'
```
Comment (AlexNisnevich) [inline src/common/components/game/VolumeControl.tsx]: Hmm, any way to merge this and the above style?

File: src/common/components/game/VolumeControl.tsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+import IconButton from '@material-ui/core/IconButton';
+import Icon from '@material-ui/core/Icon';
+import { withStyles, WithStyles } from '@material-ui/core/styles';
+import Slider from '@material-ui/lab/Slider';
+
+import { MAX_Z_INDEX } from '../../constants';
+
+import Tooltip from '../Tooltip.js';
+
+const styles = {
```
Comment (AlexNisnevich) [inline src/common/components/game/VolumeControl.tsx]: I would put this as a static inside `VolumeControl`, like I did with `SetSummary`.

**Comment by AlexNisnevich** [comment]
Looks good overall (minus CI issue).

But why not make the volume button also show/hide the volume slider on hover, instead of introducing a new button?

File: src/common/components/game/GameArea.js (hunk 1 above)
Comment (jacobnisnevich) [reply src/common/components/game/GameArea.js]: Fixed

**Pull request merged**